src/frontend/get-jira-configuration.ts

```diff
@@ -42,18 +42,28 @@ export async function getInstallation(client, subscription) {
 
 		return response.data;
 	} catch (err) {
+		logger.error(
```

Comment:
we need to use logger here from the request `req.log` because this logger has context information (like request id, jira hsotname, etc)

[ ] Use `req.log` instead of creating a new logger in this file

Reply:
We can pass logger to getInstallation as a parameter hence it called from different places in the code.